tools/add_triage_signature.py

```diff
@@ -27,6 +27,10 @@
 from tabulate import tabulate
 
 DEFAULT_DAYS_TO_HANDLE = 30
+CF_USER = "customfield_12319045"
```

Comment:
no other way to refer to those fields?

Reply:
Non-standard fields have these weird names

Reply:
OK no biggie